### PR TITLE
Update videos list

### PR DESCRIPTION
**Pull request by LifanPan**
Added new videos for RX and RL78 family
- How to Create a Blinky Project with Smart Configurator on e² studio (RX & RL78)
- How to Change FIT Module Version in Smart Configurator (RX)
- How to Use Symbolic Name with Smart Configurator (RX & RL78)
- How to Use CAN/LIN with Smart Configurator on e² studio (RL78)
- How to Create a LIN Project with Smart Configurator on e² studio (RL78)

Added new page for RISC-V
- Usage of RISC-V MCU Smart Configurator in IAR Embedded Workbench - Quick Start Guide
- How to Create a RISC-V Smart Configurator Project in e² studio
- Smart Configurator Feature - How to generate HEX values for Code Generator drivers
- How to Use the User Code Protection Feature in Smart Configurator
- How to Create a Blinky Project with Smart Configurator on e² studio

Added new page for RZ
- Introduction of Smart Configurator for RZ Family
- How to Create a Smart Configurator project in e² studio

Common Page will be deprecated.

Updated Video Categories

Fixed duplicate video IDs causing bug for New-Videos list.

### CODE REVIEW

File: Videos/videos_info.xml [hunk 1]
```diff
@@ -242,18 +242,53 @@
         <date>2023-06-14</date>
         <renesaslink>https://www.renesas.com/video/how-use-smart-configurator-walkthrough-settings-clocks-and-components</renesaslink>
     </info>
-    <info id="_Toc87439381" name="videoInfo" device="RX || RL78 || RH850 || COMMON" category="tutorials">
```
Comment (leonardkong) [inline Videos/videos_info.xml:245]: Q: Why is this removed?

File: Videos/videos_info.xml [hunk 2]
```diff
@@ -242,18 +242,53 @@
         <date>2023-06-14</date>
         <renesaslink>https://www.renesas.com/video/how-use-smart-configurator-walkthrough-settings-clocks-and-components</renesaslink>
     </info>
-    <info id="_Toc87439381" name="videoInfo" device="RX || RL78 || RH850 || COMMON" category="tutorials">
+    <info id="_Toc87439381" name="videoInfo" device="RX || RL78" category="features">
```
Comment (leonardkong) [inline Videos/videos_info.xml:245]: Q: What is the new definition of a feature and a tutorial?

File: Videos/videos_info.xml (hunk 1 above)
Comment (LifanPan) [reply Videos/videos_info.xml:245]: This video is removed from RH850 and Common pages as it is no longer listed in the excel sheet.

File: Videos/videos_info.xml (hunk 2 above)
Comment (LifanPan) [reply Videos/videos_info.xml:245]: I don't think there is any change to the definition of a feature and a tutorial. Previous categorization of videos was done purely by me using the naming of the videos as a gauge. For example, "How to..." videos are automatically tutorial videos.
Following the excel sheet, some "How to..." videos are re-categorized to features like the Blinky Project feature.

File: Videos/Smart Configurator Videos_RH850.htm [hunk 3]
```diff
@@ -31,6 +31,9 @@ <h1>Smart Configurator Videos for RH850 Family</h1>
                             <li style="list-style-type: none;">
                                 <a href="#features-wrapper" class="features">Basic Features</a>
                             </li>
+                            <!-- <li style="list-style-type: none;">
```
[the remark this answers is not in the document]
Comment (LifanPan) [reply Videos/Smart Configurator Videos_RH850.htm]: Removed all commented code

File: Videos/Smart Configurator Videos_RZ.htm [hunk 4]
```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (LifanPan) [reply Videos/Smart Configurator Videos_RZ.htm]: Removed file

**Review by leonardkong — APPROVED** [review]
